.github/workflows/ci.yml

```diff
@@ -53,5 +53,13 @@ jobs:
       env:
        TOXENV: INTEGRATION
       run: |
-        python -m pip install .
+        python -m pip install . --use-feature=in-tree-build
+        python test_runner.py
+
+    - name: Run Integration Test (uvloop)
+      if: matrix.os != 'windows-latest'
```

Comment:
Nice - Only bummer here is it is now serially run after non uvloop ... But we can live with it